ATTRIBUTIONS-Rust.md

```
@@ -5085,7 +5080,7 @@ limitations under the License.
 ## regex-automata - 0.4.9
 **Repository URL**: https://github.com/rust-lang/regex/tree/master/regex-automata
 **License Type(s)**: Apache-2.0 OR MIT
-### License: https://github.com/rust-lang/regex/tree/master/regex-automata/blob/HEAD/LICENSE-APACHE
+### License: https://github.com/rust-lang/regex/blob/HEAD/LICENSE-APACHE
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)